config/base/webpack.config.ts

```diff
@@ -8,7 +8,7 @@ import Config from "webpack-config";
 
 const {peerDependencies} = require("../../package.json"); // tslint:disable-line
 
-const outPath = join(__dirname, "../../test-dist");
+const outPath = join(__dirname, "../../dist");
```

Comment:
Maybe I'm confused but base shouldnt output to test-dist

Reply:
Agreed. The test config was previously the only config extension of the base config, that might be why this ended up the way it did. You'll need to make sure that the output directory for the test config still points to test-dist.

Reply:
I just tested locally with this branch and confirmed that the test files are still added to the `test-dist` directory